DataLayer/Mapper/OrderDataMapper.php

```diff
@@ -61,6 +61,7 @@ public function mapByOrder(OrderInterface $order): array
             'coupon' => $order->getCouponCode(),
             'date' => date("Y-m-d", strtotime($order->getCreatedAt())),
             'paymentType' => $this->getPaymentType($order),
+            'payment_method' => $order->getPayment()?->getMethod() ?? '',
```

Comment:
I know that this is similar to `paymentType` above. However, `paymentType` is the title of the payment method, which is, IMHO, not suitable for tracking as it can change regularly, differs by language etc.